src/main/pages/che-7/overview/proc_importing-self-signed-tls-certificates-to-browsers.adoc

```diff
@@ -9,30 +13,6 @@ This section describes how to import a root certificate authority into a web bro
 
 When a TLS certificate is not trusted, the error message *Authorization token is missing. Click here to reload page* blocks the login process. To prevent this, add the public part of the self-signed CA certificate into the browser after installing {prod-short}.
```

Comment:
should we add a link to `Exporting the self-signed CA certificate from a {prod} deployment` section in case user didn't first export the cert they need to import?

Reply:
I'm not sure -- the doc on exporting the self-signed cert is kind of limited to admins (get secret or run chectl). This part refers to going through the browser process of trusting a cert (click the cert, save it, and import it into your browser). I think the exporting procedure is no longer necessary in general.

The other side of it is importing the cert into your brower, but I don't know how much we want to document external programs.

I think @tomgeorge was working on something related.